_posts/2016-03-04-hunting-down-a-slow-rails-request.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Hunting Down a Slow Rails Request"
+date: 2016-03-04 11:03:00
+categories: ["rails", "performance"]
+author: "mauro-oto"
+---
+
+We have recently began using [Skylight](https://www.skylight.io) in production
```

Comment:
@mauro-oto I think it'd sound more natural if you said: "Recently we started using... "